### PR TITLE
restructure init scripts to add anchors

### PR DESCRIPTION
**Pull request by aptalca**
This PR modifies the init structure to add anchors for the baseimage init steps so mods and other customizations can easily inject things in precise locations, and that it's easier to manage for downstream images. Below is the init flow in order:

- `init-os-end` - anchor that represents the end of the root baseimage init steps (ubuntu/alpine/arch/fedora)
- `init-kasmvnc` - anchor that represents the beginning of the kasmvnc baseimage init steps
- `init-nginx`
- `init-kasmvnc-config`
- `init-video`
- `init-kasmvnc-end` - anchor that represents the end of the kasmvnc baseimage init steps
- `init-config` - anchor that represents the beginning of the downstream image init steps

In a nutshell, this structure sandwiches all the kasmvnc init steps between the 2 new achors, and they run after the root baseimage init (`init-os-end`) and before the downstream image init (`init-config`)

To get the full log that shows the order of the init steps, we can pass the env var `S6_VERBOSITY=2`

### CODE REVIEW

**Review by thelamer — APPROVED** [review]
Thanks @aptalca
